css/index.css

```diff
@@ -62,4 +62,12 @@ body {
     margin-bottom: 5px;
     font-size: 12px;
     color: #777;
+}
+
+.EN {
```

Comment:
Add some comment, since css is a not that explicit

Reply:
Fixed with 87238d36a2b5be6dcda0c0570ff29ecb46a342f7